base/server/cmscore/src/com/netscape/cmscore/ldapconn/LdapAnonConnFactory.java

```diff
@@ -133,11 +154,15 @@ protected void init(int minConns, int maxConns, LdapConnInfo connInfo)
         if (minConns > maxConns)
             throw new ELdapException("Minimum number of connections is bigger than maximum: " + minConns + " > " + maxConns);
 
+        if (maxResults <= 0)
```

Comment:
We may want to allow 0 for no limit. See explanation below.

Reply:
Ah forgot about 0.